data/json/mutations.json

```diff
@@ -2503,7 +2502,7 @@
         "id" : "PRED3",
         "name" : "Predator",
         "points" : 3,
-        "description" : "Let's not overthink things.  You get hungry, you find prey, you kill it and you eat it.",
+        "description" : "You consider yourself something other than human and no longer empathise with them.  Combat skills are easy to learn and maintain, but your critical thinking, so characteristic to these creatures, suffers: -1 INT.",
```

Comment:
I'm not sure it's okay to show such explicit stat changes in the description (i.e. ```INT -1```), it's better to describe it verbally.